schemas/components/schemas/VpnResult.yaml

```diff
@@ -46,6 +46,12 @@ properties:
       - publicVPN
       - auxiliaryMobile
       - osMismatch
+  confidence:
+    type: string
+    description: >-
```

Comment:
Looks like [enum](https://swagger.io/docs/specification/data-models/enums/), please add if you agree